dbt_meshify/dbt_projects.py

```diff
@@ -150,9 +154,21 @@ def get_catalog_entry(self, unique_id: str) -> Optional[CatalogTable]:
         """Returns the catalog entry for a model in the dbt project's catalog"""
         return self.catalog.nodes.get(unique_id)
 
-    def get_manifest_node(self, unique_id: str) -> Optional[ManifestNode]:
+    def get_manifest_entry(self, unique_id: str) -> Optional[ManifestNode]:
         """Returns the catalog entry for a model in the dbt project's catalog"""
```

Comment:
I don't quite agree with renaming this method. This specifically is getting a `ManifestNode`, which would indicate that `get_manifest_node` would be an appropriate name.